src/main/java/billgates/view/gui/TopMenuBar.java

```diff
@@ -114,4 +146,42 @@ public void loadFile(File file) {
         // CONTINUE!!!
         System.out.println("loaded");
     }
+
+    //Function to show a description of the function of this app
+    private void showFunctions(){
```

Comment:
Small convention issue here, should be: 
```suggestion
    private void showFunctions() {
```